Unable to build v0.5 on archlinux

Hi, I'm trying to build gitsh v0.5 for AUR but make exits with errors. Here's the complete output (ruby version is 2.1.1).

```
$ makepkg                                                                                                                                                                 
==> Making package: gitsh 0.5-1 (lun 10 mar 2014, 07.06.59, CET)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found gitsh-0.5.tar.gz
==> Validating source files with sha1sums...
    gitsh-0.5.tar.gz ... Passed
==> Extracting sources...
  -> Extracting gitsh-0.5.tar.gz with bsdtar
==> Starting build()...
configure: WARNING: unrecognized options: --disable-debug, --disable-dependency-tracking
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for ruby... /bin/ruby
checking for ruby... (cached) /bin/ruby
checking for a sed that does not truncate output... /usr/bin/sed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for ruby version... ruby 2.1.1p76 (2014-02-24 revision 45161) [armv6l-linux-eabihf]
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating src/Makefile
config.status: creating lib/gitsh/Makefile
config.status: creating man/Makefile
config.status: creating vendor/Makefile
configure: WARNING: unrecognized options: --disable-debug, --disable-dependency-tracking
Making all in src
make[1]: Entering directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/src'
sed -e 's|@RUBY[@]|/bin/ruby|g' -e 's|@rubydir[@]|/usr/share/gitsh/ruby|g' -e 's|@gemsetuppath[@]|/usr/share/gitsh/gems/setup.rb|g' ./gitsh.in > gitsh
chmod +x gitsh
make[1]: Leaving directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/src'
Making all in man
make[1]: Entering directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/man'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/man'
Making all in lib/gitsh
make[1]: Entering directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/lib/gitsh'
sed -e 's|@PACKAGE_VERSION[@]|0.5|g' ./version.rb.in > version.rb
make[1]: Leaving directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/lib/gitsh'
Making all in vendor
make[1]: Entering directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/vendor'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/vendor'
make[1]: Entering directory '/home/cippaciong/build/gitsh/src/gitsh-0.5'
make[1]: Nothing to be done for 'all-am'.
make[1]: Leaving directory '/home/cippaciong/build/gitsh/src/gitsh-0.5'
==> Entering fakeroot environment...
==> Starting package()...
Making install in src
make[1]: Entering directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/src'
make[2]: Entering directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/src'
 /usr/bin/mkdir -p '/home/cippaciong/build/gitsh/pkg/gitsh//usr/bin'
 /usr/bin/install -c gitsh '/home/cippaciong/build/gitsh/pkg/gitsh//usr/bin'
make[2]: Nothing to be done for 'install-data-am'.
make[2]: Leaving directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/src'
make[1]: Leaving directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/src'
Making install in man
make[1]: Entering directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/man'
make[2]: Entering directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/man'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/home/cippaciong/build/gitsh/pkg/gitsh//usr/share/man/man1'
 /usr/bin/install -c -m 644 man1/gitsh.1 '/home/cippaciong/build/gitsh/pkg/gitsh//usr/share/man/man1'
make[2]: Leaving directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/man'
make[1]: Leaving directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/man'
Making install in lib/gitsh
make[1]: Entering directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/lib/gitsh'
make[2]: Entering directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/lib/gitsh'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/home/cippaciong/build/gitsh/pkg/gitsh//usr/share/gitsh/ruby/gitsh'
 /usr/bin/install -c -m 644 cli.rb colors.rb completer.rb environment.rb git_command.rb git_repository.rb history.rb internal_command.rb interpreter.rb module_delegator.rb parser.rb program_name.rb prompter.rb readline_blank_filter.rb shell_command.rb transformer.rb tree.rb version.rb version.rb '/home/cippaciong/build/gitsh/pkg/gitsh//usr/share/gitsh/ruby/gitsh'
/usr/bin/install: will not overwrite just-created ‘/home/cippaciong/build/gitsh/pkg/gitsh//usr/share/gitsh/ruby/gitsh/version.rb’ with ‘version.rb’
Makefile:263: recipe for target 'install-dist_pkgrubyDATA' failed
make[2]: *** [install-dist_pkgrubyDATA] Error 1
make[2]: Leaving directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/lib/gitsh'
Makefile:333: recipe for target 'install-am' failed
make[1]: *** [install-am] Error 2
make[1]: Leaving directory '/home/cippaciong/build/gitsh/src/gitsh-0.5/lib/gitsh'
Makefile:327: recipe for target 'install-recursive' failed
make: *** [install-recursive] Error 1
==> ERROR: A failure occurred in package().
    Aborting...
```

--- a/lib/gitsh/Makefile.am
+++ b/lib/gitsh/Makefile.am
@@ -1,4 +1,5 @@
-dist_pkgruby_DATA = $(libfiles) version.rb
+pkgruby_DATA = version.rb
+dist_pkgruby_DATA = $(libfiles)
 CLEANFILES = version.rb
 EXTRA_DIST = version.rb.in
 

